frontend/src/pages/add-blog.tsx

```diff
@@ -122,15 +139,27 @@ function AddBlog() {
             onChange={handleInputChange}
           />
         </div>
-        <div className="mb-4">
-          <input
-            type="text"
-            name="imageLink"
-            placeholder="Image URL"
-            className="w-full p-2 rounded-lg bg-gray-50 placeholder:text-gray-800 "
-            value={formData.imageLink}
-            onChange={handleInputChange}
-          />
+        <div className="flex">
+          <div className="mb-4 flex flex-1 mr-16">
+            <input
+              type="text"
+              id="imgtext"
+              name="imageLink"
+              placeholder="Image URL"
+              className="w-full p-2 rounded-lg bg-gray-50 placeholder:text-gray-800 "
+              value={formData.imageLink}
+              onChange={handleInputChange}
+            />
+          </div>
+          <button
+            type="button"
+            className="bg-black text-white items-center justify-center text-base p-2 rounded-lg hover:bg-gray-800 w-full md:w-fit flex flex-2"
+            onClick={() => {
```

Comment:
Here can we have it's height same to the image url, don't mind I'm a little picky; because according to the UX, this button out shines more than the create blog button.

Reply:
Fixed